news/2022-07-08-the-roundtable.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "The Roundtable"
+date: 2022-07-08 10:00:00 +0000
+---
+
+16 osu! warriors battle in the inaugural edition of **The Roundtable**, osu!'s first major in-person invitational tournament!
+
+![](/wiki/shared/news/2022-07-08-the-roundtable/roundtable_banner.png)
+
+While LAN events are nothing new to the osu! scene, [**The Roundtable**](/wiki/Tournaments/The_Roundtable) takes a different approach.
+
+This **3 day** event from July 8-10 features **3 different tournaments**, all streamed from [BTMC](https://osu.ppy.sh/users/3171691)'s [Twitch channel](https://twitch.tv/BTMC) with comfy couch commentary and personal interactions with everyone involved!
+
+## Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| [16 player battle royale](/wiki/Tournaments/The_Roundtable/16_player_battle_royale) | 2022-07-06 (21:00 UTC +0) |
+| [2v2 team tandem tournament](/wiki/Tournaments/The_Roundtable/2v2_team_tandem_tournament) | 2022-07-07 (18:00 UTC +0) |
+| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) | 2022-07-07 (22:00 UTC +0) |
+| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) (top 8) | 2022-07-08 (17:30 UTC +0) |
```

Comment:
Are the dates wrong here? Above it says july 8-10 but this is 6-8, and mostly in the past..

Reply:
Indeed, they should be 8-10th July

Reply:
8-10 is right yeah